### ISSUE TITLE
Incorrect fix mode in LongLine rule

### ISSUE THREAD

**Issue by Arrgentum** [saveourtool/diktat#1385]
## Test

maximum linelength = 151

### Initial code:
```kotlin
val elem = (hasExplicitNotUnitReturnType || isFunWithExpressionBody && !hasExplicitUnitReturnType && hasNotExpressionBodyTypes) && !hasReturnKdoc && !isReferenceExpressionWithSameName
```

### Expected behavior
```kotlin
val elem = (hasExplicitNotUnitReturnType || isFunWithExpressionBody && !hasExplicitUnitReturnType && hasNotExpressionBodyTypes) && !hasReturnKdoc && 
        !isReferenceExpressionWithSameName
```

### Observed behavior
```kotlin
val elem = 
    (hasExplicitNotUnitReturnType || isFunWithExpressionBody && !hasExplicitUnitReturnType && hasNotExpressionBodyTypes) && !hasReturnKdoc && !isReferenceExpressionWithSameName
```

## Environment information
* diktat version:  1.1.0
* kotlin version 1.6
